config/m3aggregator.yaml

```diff
@@ -57,6 +57,16 @@ msgpack:
     iteratorPool:
       size: 4096
 
+runtimeOptions:
+  kvConfig:
+    namespace: /m3aggregator
+    environment: production
+  writeValuesPerMetricLimitPerSecondKey: write-values-per-metric-limit-per-second
```

Comment:
nit: can this be write-frequency( or qps)(-limit)-per-metric-per-second?

Reply:
But we are writing metric values, no? As mentioned above, the idea of "frequency" is already captured by "value per metric per second"?